CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## Main
 
 * [6444](https://github.com/grafana/loki/pull/6444) **aminesnow** Add TLS config to query frontend.
-* [6435](https://github.com/grafana/loki/pull/6430) **MichelHollands**: Remove the `whole-stream-deletion` mode.
+* [6435](https://github.com/grafana/loki/pull/6435) **MichelHollands**: Remove the `whole-stream-deletion` mode.
```

Comment:
Do you mind also moving it to the Unreleased/All Changes/Loki/Changes section? The entries listed under this "Main" section are wrong I believe; you can even see that some of them are duplicates of older versions..

Reply:
I've added some lines to the changes section, up until the next one that is in the list as well. However there seems to be a lot missing?

Reply:
Yep, a lot of them are under Main. I'm not sure at which point we're planning to fix them, but I think it will be soon since we're close to release 2.6